README.md

```diff
@@ -14,243 +14,222 @@
+
+#### ZONES
+Specify one or more irrigation zones here. The component calculates irrigation duration per zone, depending on size, throughput, state, module and mapping. A zone can be _disabled_ (so it doesn't do anything), _automatic_ or _manual_. If in automatic, duration is automatically calculated. If in manual, you specify the duration yourself. If disabled, the zone is not included in any calculations.
+
+> **When entering any values in the configuration of this component, keep in mind that the component will expect inches, sq ft, gallons, gallons per minute, or mm, m<sup>2</sup>, liters, liters per minute respectively depending on the settings in Home Assistant (imperial vs metric system).
+
+- Name: The name of your zone, e.g. garden
+- Size: The size of this zone (m<sup>2</sup> or sq ft)
+- Throughput: The flow of this zone (liter/minute or ???/minute)
+- Bucket: You can manipulate this calculated value.
+- Lead Time: In seconds. Time needed to warm up your irrigation system, e.g. time to establish a connection or start a pump etc.
+
+You can Update all zones and Callculate all.
+
+**Per Zone Settings**
+You can change any value mentioned before. Additionaly there are some more options.
+
+- State: Wether Updating and Calculation of that zone should be enabled in mode 'Automatic' or 'Manual' or 'Disabled'
+- Module: Choose the calculation module (see below) shall be used for that zone to calculate irrigation duration
+- Mapping: Which sensor mapping (see below) shall be used for that zone
+- Bucket: Either calculated or manualy set. Bucket >=0 no irrigation is necesarry, bucket <0 irrigation is necesarry.
+- Multiplier: Multiplies the duration of irrigation
+- Duration: Either calculated or manualy set
+
+Below each zone there are some buttons to update, calculate or delete that zone.
+
+#### MODULES
+Add one or more modules that calculate irrigation duration. Each module comes with its own configuration and can be used to calculate duration for one or more zones. Modules can't be deleted if they are used by one or more zones.
+
+- 0:PyETO: Calculate duration based on the FAO56 calculation from the PyETO library. 
+  - If you set PyETO to not estimate, it will look for a solar radiation sensor in the mapping and will use that value. If there is none, it will use OWMs value (assuming you have OWM configured).
+If you let PyETO to estimate from temperature or sun hours, it will not ask OWM for a solar radiation value nor will it ask a sensor for a solar radiation value, even if you configured it in the mapping.
+  - Coastal: If the location you are tracking is situated on or adjacent to coast of a large land mass or anywhere else where air masses are influenced by a nearby water body, enable this setting.
+  - Solrad behaviour: Should solar radiation estimated from temperature or sun hours or disabled 
+  - Forecast days: How many forecast days taken into account
+- 1:Static: 'Dummy' module with a static configurable duration.
```

Comment:
this should be static configurable netto precipitation

Reply:
done